incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -44,9 +44,16 @@ To install the plugin (Linux-only), first switch to a directory on your `PATH`
 # shown below, as of August 2020.
 HNC_VERSION=v0.5.1
 
+# Seletct the OS and architecture type (from v0.5.2) where you run the plugin. 
+# Currently, you can choose "linux_amd64" or "darwin_amd64" (Mac).
+HNC_PLATFORM=linux_amd64
+
 # Download the plugin
 curl -L https://github.com/kubernetes-sigs/multi-tenancy/releases/download/hnc-${HNC_VERSION}/kubectl-hns -o ./kubectl-hns
 
+# Download the plugin (from v0.5.2)
+curl -L https://github.com/kubernetes-sigs/multi-tenancy/releases/download/hnc-${HNC_VERSION}/kubectl-hns_${HNC_PLATFORM} -o ./kubectl-hns
+
```

Comment:
Firstly, there's a typo ("Select", not "Seletct").

Secondly, this has a 0.5.2 instruction, then an instruction that doesn't appear to care which version it's for (but actually does), then another 0.5.2 instruction. How about grouping them:

```
# HNC v0.5.1 or earlier: download the plugin (Linux only)
curl ...

# HNC v0.5.2 or later: set the platform and download the plugin. The support platforms are:
# * linux_amd64 (Linux on Intel/AMD)
# * darwin_amd64 (MacOS on Intel)
HNC_PLATFORM=linux_amd64
curl ...
```